src/components/Pagination/__tests__/Item.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Pagination } from '../'
+
+describe('<Pagination.Item />', () => {
+  it('renders children', () => {
+    const $ = shallow(
+      <Pagination.Item to="/">
```

Comment:
`Pagination.Item` isn't expecting a `to` prop.

Reply:
done